learningequality/studio
# Topic to folder

## Summary
### Description of the change(s) you made
Changed all instances of **Topic** to **Folder** that are user facing, by chaging trs objects and createTranslator instances. 
Checked and tested everything I could find using the admin account. Changed 1 test to indicate the new changes. 


### Manual verification steps performed
1. Went trough the whole applicaiton with an admin account several times to check any place topics might remain.
2. Ran the test suite multiple time to ensure everything is working properly


### Does this introduce any tech-debt items?
Whilst changing Topic => Folder is a user facing thing now, we need to change topic to folder on a code level so that there is no future confusion


## Comments
Since this is the first major user facing change I have made, this has to be tested properly by someone with more experience with the project.


### Contributor's Checklist


PR process:



Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
